GCRCatalogs/cosmodc2.py

```diff
@@ -436,8 +436,8 @@ def _generate_quantity_modifiers(self):
             'stellar_mass':  'totalMassStellar',
             'stellar_mass_disk':        'diskMassStellar',
             'stellar_mass_bulge':       'spheroidMassStellar',
-            'size_disk_true':           'morphology/diskMajorAxisArcsec',
-            'size_bulge_true':          'morphology/spheroidMajorAxisArcsec',
+            'size_disk_true':           'morphology/diskHalfLightRadiusArcsec',
+            'size_bulge_true':          'morphology/spheroidHalfLightRadiusArcsec',
```

Comment:
Are these backward compatible?  How about the older files that still use `morphology/diskMajorAxisArcsec` and `morphology/spheroidMajorAxisArcsec`

Reply:
I decided to remove the 2 configuration files that corresponded to catalogs that were not in the shared space. 
The changes for size_disk_true and size_bulge_true are actually a bug fix, although due to the bug, the numerical value will not change. Let me explain: previously diskMajorAxisArcsec and spheroidMajorAxisArcsec were set to the half light radius, which was incorrect. ie the major axes and the sizes were all set to the same value.  The size should have been set to the half light radius.  This PR corrects that bug.  The native major and minor axes are still incorrect, but that is another issue that needs to be more widely discussed.  I hope that clarifies the situation.  Please let me know if you have any questions.

Reply:
Thanks for the explanation. That sounds alright and we should just include a brief summary of this bug fix in the upcoming release so that people are aware of it.